tools/VerifyCopyrightHeaders.cmd

```diff
@@ -1,14 +1,6 @@
 @echo off
 SETLOCAL
 
-IF /I "%1" == "-help" GOTO Help
-IF /I "%1" == "--help" GOTO Help
-
 powershell -ExecutionPolicy Unrestricted -NoLogo -NoProfile %~dpn0.ps1 %*
-GOTO TheEnd
-
-:Help
-powershell -ExecutionPolicy Unrestricted -NoLogo -NoProfile -c Get-Help %~dpn0.ps1 -full
```

Comment:
How do I get usage help?

`VerifyCopyrightHeaders --help` ?

`VerifyCopyrightHeaders -?` ?

Please don't say `VerifyCopyrightHeaders -c Get-Help`

Reply:
There is no -help.  You run the script.  If it finds issues, it tells you want to do next.